x-pack/plugins/spaces/server/saved_objects/spaces_saved_objects_client.ts

```diff
@@ -378,4 +380,34 @@ export class SpacesSavedObjectsClient implements SavedObjectsClientContract {
       namespace: spaceIdToNamespace(this.spaceId),
     });
   }
+
+  /**
+   * Opens a Point In Time (PIT) against the indices for the specified Saved Object types.
+   * The returned `id` can then be passed to `SavedObjects.find` to search against that PIT.
+   *
+   * @param {string|Array<string>} type
+   * @param {object} [options] - {@link SavedObjectsPointInTimeOptions}
+   * @property {string} [options.keepAlive]
+   * @property {string} [options.preference]
+   * @returns {promise} - { id: string }
+   */
+  async openPointInTimeForType(
+    type: string | string[],
+    options: SavedObjectsOpenPointInTimeOptions = {}
+  ) {
+    throwErrorIfNamespaceSpecified(options);
+    return await this.client.openPointInTimeForType(type, {
+      ...options,
+      namespace: spaceIdToNamespace(this.spaceId),
+    });
+  }
+
+  /**
+   * Closes a Point In Time (PIT) by ID. This simply proxies the request to ES
+   * via the Elasticsearch client, and is included in the Saved Objects Client
+   * as a convenience for consumers who are using `openPointInTimeForType`.
+   */
+  async closePointInTime(id: string, options?: SavedObjectsClosePointInTimeOptions) {
+    return await this.client.closePointInTime(id, options);
```

Comment:
Missing the expected space-constraints here (and associated tests):

```suggestion
    throwErrorIfNamespaceSpecified(options);
    return await this.client.closePointInTime(id, {
      ...options,
      namespace: spaceIdToNamespace(this.spaceId),
    });
```

Reply:
done